ide-support/revsaveasandroidstandalone.livecodescript

```diff
@@ -1017,6 +1019,16 @@ end revSaveAsMobileStandaloneMain
 
 ################################################################################
 
+private function escapeLabel pLabel
+   replace "&" with "&amp;" in pLabel
+   replace "@" with "\@" in pLabel
+   replace "?" with "\?" in pLabel
+   replace "<" with "&lt;" in pLabel
+   replace "'" with "\'" in pLabel
+   replace quote with "\" & quote in pLabel
+   return pLabel
```

Comment:
Does `">"` need replacing with `"&gt;` also?

Reply:
Thanks @livecodeali, just added it